packages/libs/src/sdk/services/EntityManager/types.ts

```diff
@@ -92,7 +93,7 @@ export type ManageEntityOptions = {
   /**
    * The numeric user id
    */
-  userId: number
+  userId?: number
```

Comment:
Unsure if downloads are gated on being signed in at a protocol level, so made user ID optional